Sources/TuistGenerator/Linter/SchemeLinter.swift

```diff
@@ -92,6 +92,13 @@ extension SchemeLinter {
                     issues.append(missingExpandVariablesTargetIssue(missingTargetName: target.name, schemaName: scheme.name))
                 }
             }
+            if let runAction = scheme.runAction,
+               let target = runAction.expandVariableFromTarget
+            {
+                if !targetNames.contains(target.name) {
+                    issues.append(missingExpandVariablesTargetIssue(missingTargetName: target.name, schemaName: scheme.name))
+                }
+            }
```

Comment:
There's one more lint rule we can add to help catch another common issue:

For a project with the following targets:

- App
- Framework1

And a Scheme specified 

Build Action:
- targets: [App]

Run action:
- Executable: App
- Expand Variables From: Framework1

Xcode will not accept setting `Framework1` as target you can expand variables from if it's not listed as one of the build targets.